test/e2e/artifact_test.go

```diff
@@ -158,6 +160,47 @@ var _ = Describe("Podman artifact", func() {
 		Expect(a.Name).To(Equal(artifact1Name))
 	})
 
+	It("podman artifact push and pull with authorization", func() {
```

Comment:
name says push and pull but this test only ever tries push

Reply:
Fixed, thanks!